waiter/config-shell.edn

```diff
@@ -26,6 +26,12 @@
  ;; Set the port:
  :port #config/env-int "WAITER_PORT"
 
+ ; ---------- Token Storage ----------
+
+ :kv-config {:kind :persistent
```

Comment:
Maybe `:kind` could be `:file` or `:file-based`? `:persistent` seems too general

Reply:
okay